protostar/protostar_cli_test.py

```diff
@@ -28,94 +30,92 @@ def version_manager_fixture(mocker: MockerFixture, git_version: str):
     return version_manager
 
 
+@pytest.fixture(name="logger")
+def logger_fixture(mocker: MockerFixture):
+    return mocker.MagicMock()
```

Comment:
Do we want to hide logs from tests? I have personally used Protostar output a lot while debugging integration tests

Reply:
+1